site/build.gradle

```diff
@@ -67,6 +67,13 @@ task javadoc(type: Javadoc,
         addBooleanOption('Xdoclint:all').value = true
         addBooleanOption('Xdoclint:-missing').value = true
 
+        //add support for java8u121 "New --allow-script-in-comments option for javadoc"
+        //ref: http://www.oracle.com/technetwork/java/javase/8u121-relnotes-3315208.html
+        if (javax.tools.ToolProvider.getSystemDocumentationTool()
+                .isSupportedOption("--allow-script-in-comments")) {
+            addBooleanOption('-allow-script-in-comments').value = true
```

Comment:
Let me take care of this in a follow-up PR. Let's revert this for now so that the build passes at Travis CI.